frontend-public/src/components/common/NullScreen.js

```diff
@@ -29,8 +29,14 @@ const NullScreen = (props) => (
     )}
     {props.type === "404" && (
       <div className="no-nav-bar">
-        <h3>404 - Page not found</h3>
-        <Link to={routes.HOME.route}>Go Home</Link>
+        <h1>Uh Oh!</h1>
+        <p>
+          We can't seem to find the page you're looking for. It may have moved, or no longer exists.
```

Comment:
Grammar feedback here. The first sentence ends with a preposition, and the second sentence is technically an incorrect use of a conjunction.
We could avoid these with a single sentence, like:
`The page you're looking for has moved or no longer exists`

Or a small adjustment to the existing two sentences:
`The page you're looking for can't be found. It may have moved, or it no longer exists.`

Reply:
Thanks for the feedback! will update with the second option.